src/poliastro/plotting/_base.py

```diff
@@ -159,6 +181,28 @@ def _plot(self, orbit, *, label=None, color=None, trail=False):
             coordinates, orbit.r, label=label, colors=colors, dashed=True
         )
 
+    def _anim(self, orbit, *, label=None, color=None, trail=False):
+        colors = self._get_colors(color, trail)
+
+        self.set_attractor(orbit.attractor)
+        from poliastro.twobody import propagation
+
+        orbit = orbit.change_plane(self.plane)
+
+        label = generate_label(orbit.epoch, label)
+        time_laps = [i for i in range(800)]
+        tofs = time_laps * u.day * 2
+        k = propagation.farnocchia(orbit.attractor.k, orbit.r, orbit.v, tofs)[0]
```

Comment:
Why do we need to explicitly call the propagator here, instead of `propagation.propagate` and that's it? The propagation method should be an implementation detail, even at this level.

Reply:
for some reason propagation.propagate is not working in animation as expected and further, it is also using farnocchia as default. therefore used the propagator directly. 
testing it ,will soon update this pull request with it. Thanks for review

Reply:
it definitely _should_ work, so please write here the full traceback if it doesn't. Thanks!

Reply:
it is 👍🏼